notification/src/handler/notification/notification.handler.js

```diff
@@ -200,7 +213,21 @@ async function calculateUpdateActionsForPayment(payment, notification, logger) {
         )
       )
     }
+    const paymentKey = payment.key
+
+    if (
+      notificationRequestItem.eventCode === 'AUTHORISATION' &&
+      JSON.parse(notificationRequestItem.success) === true && // convert 'success' from string to boolean type
```

Comment:
Please check and update your code: https://stackoverflow.com/a/7833897

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1049/commits/c4f178c8b30e148005003cd577e1071c4b01e861